pkg/controller/migmigration/restore.go

```diff
@@ -78,15 +118,23 @@ func (t Task) getRestore() (*velero.Restore, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Find proper restore to return
 	if len(list.Items) > 0 {
-		return &list.Items[0], nil
+		for i, restore := range list.Items {
+			if restore.Annotations[copyBackupRestoreAnnotationKey] != "" && copyRestore {
```

Comment:
Will `restore.Annotations[copyBackupRestoreAnnotationKey]` return nil in any case?